scalikejdbc-interpolation/src/main/scala/scalikejdbc/QueryDSLFeature.scala

```diff
@@ -475,11 +475,12 @@ trait QueryDSLFeature { self: SQLInterpolationFeature with SQLSyntaxSupportFeatu
   case class InsertSQLBuilder(override val sql: SQLSyntax) extends SQLBuilder[UpdateOperation] {
 
     def columns(columns: SQLSyntax*): InsertSQLBuilder = this.copy(sql = sqls"${sql} (${sqls.csv(columns: _*)})")
+    @deprecated(message = "values is not type safe. use `namedValues` instead.", since = "2.4.0")
```

Comment:
Are there other good way how protect the type conversion fail instead of deprecated annotation ?

Does it throw an exception at StatementExecutor?